docs/pages/review/share-previews-with-your-team.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: Share previews with your team
+description: Share previews of your changes with your team by publishing updates on branches.
+---
+
+import { ContentSpotlight } from '~/ui/components/ContentSpotlight';
+import { Terminal } from '~/ui/components/Snippet';
```

Comment:
Would love to link this article from https://docs.expo.dev/eas-update/develop-faster/ and add an image in there

Reply:
We talked about this on a call, so posting here for posterity: I talked to quin and shared that I am going to work on the guides for updates and make certain we have a better and more complete doc for previewing updates with your team.